android/src/main/java/com/pdftron/pdftronflutter/FlutterDocumentActivity.java

```diff
@@ -141,7 +142,7 @@ public static void openDocument(Context packageContext, Uri fileUri, String pass
             intentBuilder.usingNavIcon(navIconId);
             intentBuilder.usingConfig(config);
             intentBuilder.usingNewUi(true);
-            packageContext.startActivity(intentBuilder.build());
+            packageContext.startActivity(intentBuilder.build().addFlags(Intent.FLAG_ACTIVITY_NEW_TASK));
```

Comment:
any particular reason for this line? if from a documentation, please link to the documentation

Reply:
Without that flag, we would get an error that I believe said "Calling startActivity() from outside of an Activity context"